packages/11ty/_plugins/transforms/outputs/pdf/write.js

```diff
@@ -48,6 +48,11 @@ module.exports = (eleventyConfig) => {
       asset.setAttribute('src', trimLeadingSlash(src))
     })
 
+    document.querySelectorAll('[data-background-image]').forEach((element) => {
+      const { backgroundImage } = element.dataset
+      if (backgroundImage) element.style.backgroundImage = `url('${trimLeadingSlash(backgroundImage)}')`
+    })
```

Comment:
@anderspollack were we to execute this transform for all `style` background images we would not need to duplicate `data-background-image` attribute;
```suggestion
    document.querySelectorAll('[style*='background-image']').forEach((element) => {
      const { backgroundImage } = element.style
      if (backgroundImage) element.style.backgroundImage = `url('${trimLeadingSlash(backgroundImage)}')`
    })
```